### ISSUE TITLE
Keyboard Shortcut Settings

### ISSUE THREAD

**Issue by joaquimrocha** [kubernetes-sigs/headlamp#3501]
## Is your feature request related to a problem? Please describe the impact that the lack of the feature requested is creating.

We should allow users to configure the shortcut that activates the search, as it can be a highly personal choice to start search with a slash, or a ctrl+p, or something else. Same for the shortcut that shows the cluster chooser (Ctrl+Shift+L in a cluster view).
Since we should also have more shortcuts coming soon, maybe we could make this functionality beyond the search itself.

## Describe the solution you'd like

We have a Shortcuts section in the application settings that opens a dialog in which we can configure any shortcuts the app recognizes. This shortcuts dialog should also be triggered from the global search, so it would really be easy to quickly change the settings for it.

## What users will benefit from this feature?

Users who want to have more flexibility in configuring Headlamp to fit their needs.

## Additional context

We need design for this.

**Comment by illume** [comment]
Describing the shortcuts somewhere is also needed.

**Comment by alokdangre** [comment]
I would like to make a pr, using the headlamps, current design layout
